src/VirtualClient/VirtualClient.Actions/Furmark/FurmarkMetricParser2.cs

```diff
@@ -0,0 +1,167 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace VirtualClient.Actions
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using System.Text;
+    using System.Text.RegularExpressions;
+    using System.Xml.Linq;
+    using VirtualClient.Common.Contracts;
+    using VirtualClient.Contracts;
+
+    /// <summary>
+    ///  Parser for FURMARK XML output document.
+    /// </summary>
+    public class FurmarkMetricsParser2 : MetricsParser
```

Comment:
This can be renamed to something more informative about the log being parsed, like FurmarkMonitorMetricsParser